composer.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Why do we need a composer file here?

If we need it, worth just writing this explanation into the description attribute or somewhere else in this file to explain to future generations

Reply:
Per the description in this PR:
> composer.json is needed so that cow can install the docs and use them to get commits for the changelog and to commit the changelog.

It also means we already have it checked out during development which makes it easier to update docs as we go.

I'll update the composer.json description.

Reply:
done